wire-schema/src/main/java/com/squareup/wire/internal/protoparser/RpcElement.java

```diff
@@ -62,57 +58,14 @@ public final String toSchema() {
     return builder.append(";\n").toString();
   }
 
-  public static final class Builder {
-    private final Location location;
-    private String name;
-    private String documentation = "";
-    private String requestType;
-    private String responseType;
-    private final List<OptionElement> options = new ArrayList<>();
-
-    private Builder(Location location) {
-      this.location = checkNotNull(location, "location");
-    }
-
-    public Builder name(String name) {
-      this.name = checkNotNull(name, "name");
-      return this;
-    }
-
-    public Builder documentation(String documentation) {
-      this.documentation = checkNotNull(documentation, "documentation");
-      return this;
-    }
-
-    public Builder requestType(String requestType) {
-      this.requestType = checkNotNull(requestType, "requestType");
-      return this;
-    }
-
-    public Builder responseType(String responseType) {
-      this.responseType = checkNotNull(responseType, "responseType");
-      return this;
-    }
-
-    public Builder addOption(OptionElement option) {
-      options.add(checkNotNull(option, "option"));
-      return this;
-    }
-
-    public Builder addOptions(Collection<OptionElement> options) {
-      for (OptionElement option : checkNotNull(options, "options")) {
-        addOption(option);
-      }
-      return this;
-    }
-
-    public RpcElement build() {
-      checkNotNull(name, "name");
-      checkNotNull(requestType, "requestType");
-      checkNotNull(responseType, "responseType");
-
-      return new AutoValue_RpcElement(location, name, documentation, requestType, responseType,
-          ImmutableList.copyOf(options));
-    }
+  @AutoValue.Builder
+  public abstract static class Builder {
```

Comment:
These can all be interfaces if I remember correctly. Save you from typing `abstract static class` and `public abstract` everywhere.